### PR TITLE
removed inputContainer, sectionContainer, and dropdownContainer from FilterSearch CSS class

### PR DESCRIPTION
**Pull request by juliannzhou**
removed inputContainer, sectionContainer, and dropdownContainer from FilterSearch CSS class; no changes are made to the styling.

J=SLAP-2167
TEST=auto

### CODE REVIEW

File: src/components/FilterSearch.tsx [hunk 1]
```diff
@@ -202,4 +190,4 @@ function getScreenReaderText(sections: {
   });
   screenReaderText = screenReaderPhrases.join(' ');
   return screenReaderText;
-}
+}
```
Comment (nmanu1) [inline src/components/FilterSearch.tsx:193]: nit: I think we want to keep the single newline at the end

**Comment by oshi97** [comment]
This branch is 8 commits behind main, could you merge or rebase?

File: src/components/FilterSearch.tsx [hunk 2]
```diff
@@ -18,21 +18,16 @@ import { renderAutocompleteResult, AutocompleteResultCssClasses } from './utils/
...
+  optionsContainer?: string
 }
 
 const builtInCssClasses: Readonly<FilterSearchCssClasses> = {
   container: 'mb-2',
   label: 'mb-4 text-sm font-medium text-neutral-dark',
-  dropdownContainer: 'absolute z-10 shadow-lg rounded-md border border-gray-300 bg-white pt-3 pb-1 px-4 mt-1',
-  inputElement: 'text-sm bg-white outline-none h-9 w-full p-2 rounded-md border border-gray-300 focus:border-primary text-neutral-dark placeholder:text-neutral',
-  sectionContainer: 'pb-2',
+  inputElement: 'c outline-none h-9 w-full p-2 rounded-md border border-gray-300 focus:border-primary text-neutral-dark placeholder:text-neutral',
```
[the remark this answers is not in the document]
Comment (juliannzhou) [reply src/components/FilterSearch.tsx]: I did not pull before making the changes so the repo wasn't up to date. I've rebased and it should be the same now.

File: src/components/FilterSearch.tsx [hunk 3]
```diff
@@ -18,21 +18,16 @@ import { renderAutocompleteResult, AutocompleteResultCssClasses } from './utils/
 export interface FilterSearchCssClasses extends AutocompleteResultCssClasses {
   container?: string,
   label?: string,
-  dropdownContainer?: string,
   inputElement?: string,
-  sectionContainer?: string,
   sectionLabel?: string,
   focusedOption?: string,
   optionsContainer?: string,
```
Comment (nmanu1) [inline src/components/FilterSearch.tsx:24]: nit: remove the extra comma at the end

**Comment by alextaing** [comment]
I think you meant to say `inputContainer` instead of `labelContainer` in your PR title

**Pull request merged**